src/System.Management.Automation/security/SecureStringHelper.cs

```diff
@@ -254,48 +252,46 @@ internal static EncryptionResult Encrypt(SecureString input, byte[] key, byte[]
             Utils.CheckSecureStringArg(input, "input");
             Utils.CheckKeyArg(key, "key");
 
-            byte[] encryptedData = null;
-            MemoryStream ms = null;
-            ICryptoTransform encryptor = null;
-            CryptoStream cs = null;
-
             //
             // prepare the crypto stuff. Initialization Vector is
             // randomized by default.
             //
-            Aes aes = Aes.Create();
-            if (iv == null)
-                iv = aes.IV;
-
-            encryptor = aes.CreateEncryptor(key, iv);
-            ms = new MemoryStream();
-
-            using (cs = new CryptoStream(ms, encryptor, CryptoStreamMode.Write))
+            using (Aes aes = Aes.Create())
             {
+                if (iv is null)
+                {
+                    iv = aes.IV;
+                }
+
                 //
                 // get clear text data from the input SecureString
                 //
                 byte[] data = GetData(input);
+                try
+                {
+                    using (ICryptoTransform encryptor = aes.CreateEncryptor(key, iv))
+                    using (var sourceStream = new MemoryStream(data))
+                    using (var encryptedStream = new MemoryStream())
+                    {
```

Comment:
We could use new syntax and remove the nesting (here, above and below) since we reformat the code.

Reply:
I don't feel this is necessary.